datadriven/src/sgpp_datadriven.hpp

```diff
@@ -33,6 +33,7 @@
 #include <sgpp/datadriven/algorithm/DBMatDMSChol.hpp>
 #include <sgpp/datadriven/algorithm/DBMatDMSDenseIChol.hpp>
 #include <sgpp/datadriven/algorithm/DBMatDMSOrthoAdapt.hpp>
+#include <sgpp/datadriven/algorithm/DBMatDMS_SMW.hpp>
```

Comment:
What about DBMatOnlineDE_SMW.hpp?

Reply:
I thought it is enough to add DBMatDMS_SMW.hpp, as DBMatOnlineDE_SMW.hpp is contained in there. I added DBMatOnlineDE_SMW.hpp explicitly now.